server/core/region.go

```diff
@@ -617,6 +617,10 @@ func GenerateRegionGuideFunc(enableLog bool) RegionGuideFunc {
 			if len(region.GetPeers()) != len(origin.GetPeers()) {
 				saveKV, saveCache = true, true
 			}
+			if len(region.GetBuckets().GetKeys()) != len(origin.GetBuckets().GetKeys()) {
```

Comment:
When the `EnableRegionBucket` is false, should we skip it?

Reply:
The information of buckets should clear( update cache and  save disk) if `EnableRegionBucket ` is false.

Reply:
Can we add metrics about keys changed? then we can know the change rate.

Reply:
I think we don't need to add some metrics for it, it only is used. to clear the regon bucket info.

Reply:
why? here save kv and save cache is true, it will write to storage. I wonder if the change is frequent.

Reply:
If transfer bucket switch from on to off,  region will not inherit the old, so it will clear bucket information once.